sharktank/sharktank/layers/paged_attention.py

```diff
@@ -28,12 +28,31 @@
     QuantizerTensor,
     PlanarQuantizedTensor,
     StaticScaledQuantizer,
+    TensorScaledLayout,
 )
 from sharktank import ops, kernels
 
 __all__ = ["PagedAttention"]
 
 
+def cachify(tensor):
+    if isinstance(tensor, PlanarQuantizedTensor):
+        return tensor.unpack()._qs
+    return tensor
+
+
+def uncachify(tensor, quantizer):
```

Comment:
You can just do:
```
if quantizer is not None:
    return quantizer.quantize(tensor)
return tensor
```

Reply:
https://github.com/nod-ai/shark-ai/blob/3a8ac26e8e577dd28a6ed9812ade212f3912f638/sharktank/sharktank/types/quantizers.py#L153